test/unit/utils-specs.ts

```diff
@@ -36,22 +29,22 @@ describe('listChildrenProcessIds', function () {
 
   it('should return empty array for no output', async function () {
     sandbox.stub(teen_process, 'exec');
-    teen_process.exec.returns({stdout: '', stderr: ''});
+    (teen_process.exec as any).returns({stdout: '', stderr: ''});
     const result = await listChildrenProcessIds(1234);
-    result.should.eql([]);
+    expect(result).eql([]);
   });
 
   it('should return valid array of process ids', async function () {
     sandbox.stub(teen_process, 'exec');
-    teen_process.exec.returns({stdout: EXAMPLE_PS_OUTPUT, stderr: ''});
+    (teen_process.exec as any).returns({stdout: EXAMPLE_PS_OUTPUT, stderr: ''});
     const result = await listChildrenProcessIds(2412);
-    result.should.eql(['8131', '2434']);
+    expect(result).eql(['8131', '2434']);
   });
 
   it('should return empty array for no matches', async function () {
     sandbox.stub(teen_process, 'exec');
-    teen_process.exec.returns({stdout: EXAMPLE_PS_OUTPUT, stderr: ''});
+    (teen_process.exec as any).returns({stdout: EXAMPLE_PS_OUTPUT, stderr: ''});
```

Comment:
I assume we should be using the result of the above stub to avoid conversation to any